### PR TITLE
EOS-16310: Message bus  changes for SSPL Test code

### PR DESCRIPTION
**Pull request by sandeepsgt**
Signed-off-by: Sandeep Anjara <sandeep.anjara@seagate.com>

## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Please add Problem statement here...
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

File: sspl_test/rabbitmq_start_checker [hunk 1]
```diff
@@ -29,56 +29,19 @@ import random
 test_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 os.sys.path.append(os.path.join(test_path))
 from sspl_test.framework.utils.conf_utils import Conf, SSPL_TEST_CONF
+from cortx.utils.message_bus import MessageBus
```
Comment (cdeshmukh) [inline sspl_test/rabbitmq_start_checker]: rename the file too to avoid rmq name in it

Comment (sandeepsgt) [reply sspl_test/rabbitmq_start_checker]: Done

**Review by cdeshmukh — APPROVED** [review]

File: sspl_test/sspl_start_checker [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import socket
+
+# Adding sspl and sspl_test path
+test_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+os.sys.path.append(os.path.join(test_path))
+from cortx.utils.message_bus import MessageBus
+from cortx.utils.message_bus import MessageConsumer
+
+
+def process_msg(body):
```
Comment (cdeshmukh) [inline sspl_test/sspl_start_checker:31]: what this file is mainly for ? usecase

**Review by cdeshmukh — APPROVED** [review]

File: sspl_test/sspl_start_checker (hunk 2 above)
Comment (sandeepsgt) [reply sspl_test/sspl_start_checker:31]: This file is for waiting till SSPL adds `SSPL-LL service has started successfully` message into message bus, This message indicated that SSPL is initialized successfully and ready to accept requests. After that tests cases can be executed

**Review by thavanathan — APPROVED** [review]

**Pull request merged**